// ======================================================================
//
// Copyright (c) 1997 The CGAL Consortium
//
// This software and related documentation is part of an INTERNAL release
// of the Computational Geometry Algorithms Library (CGAL). It is not
// intended for general use.
//
// ----------------------------------------------------------------------
//
// release       : 
// release_date  : 1999, October 01
//
// file          : include/CGAL/boolean_operations_2.h
// package       : bops (2.2)
// source        : include/CGAL/boolean_operations_2.h
// revision      : $Revision$
// revision_date : $Date$
// author(s)     : Wolfgang Freiseisen <Wolfgang.Freiseisen@risc.uni-linz.ac.at>
//
// coordinator   : RISC Linz
//  (Wolfgang Freiseisen <wfreisei@risc.uni-linz.ac.at>)
//
// 
// ======================================================================

#ifndef CGAL_BOOLEAN_OPERATIONS_2_H
#define CGAL_BOOLEAN_OPERATIONS_2_H

#ifndef CGAL_BOPS_CONTAINER_POLYGON_2_H
#include <CGAL/bops_Container_Polygon_2.h>
#endif

#ifndef BOPS_TRAITS_2_H
#include <CGAL/bops_traits_2.h>
#endif
#ifndef CGAL_BOPS_POLYGON_2_H
#include <CGAL/bops_Polygon_2.h>
#endif
#ifndef BOPS_TRIANGLE_2_H
#include <CGAL/bops_Triangle_2.h>
#endif
#ifndef CGAL_BOPS_ISO_RECTANGLE_2_H
#include <CGAL/bops_Iso_rectangle_2.h>
#endif

#endif // CGAL_BOOLEAN_OPERATIONS_2_H
